src/main/java/frc/trigon/robot/subsystems/loader/Loader.java

```diff
@@ -0,0 +1,40 @@
+package frc.trigon.robot.subsystems.loader;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.trigon.robot.utilities.Conversions;
+
+import static frc.trigon.robot.subsystems.loader.LoaderConstants.LoaderState;
+
+public class Loader extends SubsystemBase {
+    private final static Loader INSTANCE = new Loader();
+    private final WPI_TalonSRX motor = LoaderConstants.MOTOR;
+    private LoaderState currentState;
+
+    private Loader() {
+    }
+
+    public static Loader getInstance() {
+        return INSTANCE;
+    }
+
+    /**
+     * @return the current state of the loader
+     */
+    public LoaderState getState() {
+        return currentState;
+    }
+
+    /**
+     * Sets the state of the loader.
+     *
+     * @param state the wanted state for the loader
```

Comment:
Reword. current wording doesn't make sense

Reply:
Do you disparage my wording??!?!!??!?!?!?!?!?!

Reply:
yes

Reply:
is there any problem with MY wordings?

Reply:
I misread it does make sense but doesn't explain anything. explain what the states are

Reply:
I hope you like the change, because I don't have a better one.